tests/run/relative_cimport_compare.srctree

```diff
@@ -0,0 +1,327 @@
+# mode: run
+# tag: cimport, pep489
+
+PYTHON setup.py build_ext --inplace
+PYTHON -c "import test_import"
+PYTHON -c "import test_cimport"
```

Comment:
> They look generated

Unfortunately not, they are manually created.

> somehow use code generation for the package directory structure and test files?

Is there anywhere a way how to do it or I need to do it from scratch?

Reply:
> Is there anywhere a way how to do it or I need to do it from scratch?

We don't have a specific framework to do this kind of thing

If it was generated then I'd probably suggest either:
* including the generation script in the commit (maybe Tools/), or
* putting the script as part of a srctree file and run it as step 1 (instead of including the output in the srctree file)

Given they aren't generated I don't think this requires a change. It would just have been a shame not to have the script if it already existed.

Reply:
No unfortunatelly I don't have such tool... So I am resolving the conversion. Please let me know if there is anything blocking this PR being accepted.